src/hooks/useNotifications.test.ts

```diff
@@ -355,6 +355,92 @@ describe('hooks/useNotifications.ts', () => {
     });
   });
 
+  describe('markNotificationDone', () => {
+    const id = 'notification-123';
+
+    describe('github.com', () => {
+      const accounts = { ...mockAccounts, enterpriseAccounts: [] };
+      const hostname = 'github.com';
+
+      it('should mark a notification as done with success - github.com', async () => {
+        nock('https://api.github.com/')
+          .delete(`/notifications/threads/${id}`)
+          .reply(200);
+
+        const { result } = renderHook(() => useNotifications(false));
+
+        act(() => {
+          result.current.markNotificationDone(accounts, id, hostname);
+        });
+
+        await waitFor(() => {
+          expect(result.current.isFetching).toBe(false);
+        });
+
+        expect(result.current.notifications.length).toBe(0);
+      });
+
+      it('should mark a notification as done with failure - github.com', async () => {
+        nock('https://api.github.com/')
+          .delete(`/notifications/threads/${id}`)
+          .reply(400);
+
+        const { result } = renderHook(() => useNotifications(false));
+
+        act(() => {
+          result.current.markNotificationDone(accounts, id, hostname);
+        });
+
+        await waitFor(() => {
+          expect(result.current.isFetching).toBe(false);
+        });
+
+        expect(result.current.notifications.length).toBe(0);
```

Comment:
This seems odd. If the request failed, shouldn't the notification list remain untouched, length 1? Maybe I'm missing something.

Reply:
I agree, tbh I just copy pasted the `markNotification` tests, so they were already broken before my PR 😬
 
I'm trying to figure out how to initialize a non-empty notifications array with mocked data, but without any success for now...